### DISCUSSION TITLE
Sensor34 Calibration Failed

### DISCUSSION THREAD

**Discussion by mushman92** [arendst/Tasmota#14900] [Q&A] [answered]
I setup 4 load sells to an HX711 Amp - this all connects to a NodeMCU ESP 8266. Right now just trying to calibrate the scale and I keep getting an error regarding calibration error *see image circled in red*, not really a ton of detail so im not sure where to start looking.

I did test my scale and the weight goes up with weight and down when I take the weight off *see image circled in blue* so the wiring I imagine is correct.

Since the scale appears to be working, just cant calibrate it where should I start investigating? 

Image:
![image](https://user-images.githubusercontent.com/22011134/154731606-0e0b3b24-53e0-4202-9b0d-b390b305203b.png)

EDIT - I figured it out, you need to make sure there is no weight on the scale before you start the calibration command. Sounds obvious when you think about it. lol

**Comment by ascillato2** [comment, marked as answer]
> I figured it out, you need to make sure there is no weight on the scale before you start the calibration command. Sounds obvious when you think about it. lol

Great you found the issue.

**Comment by yordanov-alexandar** [comment]
I am not sure if I should start a new discussion - I have almost the same problem with calibrating the scale. 
When I set the weight I get a change:
```
10:27:00.088 MQT: tele/scale/SENSOR = {"Time":"2023-12-25T10:27:00","HX711":{"Weight":0.011,"WeightRaw":-199,"AbsRaw":-131243}}
10:27:07.142 MQT: tele/scale/SENSOR = {"Time":"2023-12-25T10:27:07","HX711":{"Weight":0.036,"WeightRaw":-174,"AbsRaw":-114708}}
```
When I start calibrating I get this:
```
10:25:16.034 CMD: sensor34 3
10:25:16.036 SRC: WebConsole from 192.168.0.9
10:25:16.039 CMD: Grp 0, Cmd 'SENSOR', Idx 34, Len 1, Pld 3, Data '3'
10:25:16.045 MQT: stat/scale/RESULT = {"Sensor34":{"Ref":2324,"Cal":659,"Max":20000,"Item":0.0,"Change":"ON","Delta":0,"Tare":0,"AbsConvA":0,"AbsConvB":0}}
10:29:10.084 MQT: tele/scale/SENSOR = {"Time":"2023-12-25T10:29:10","HX711":{"Weight":0.011,"WeightRaw":-199,"AbsRaw":-131309}}
10:29:16.364 CMD: sensor34 2
10:29:16.366 SRC: WebConsole from 192.168.0.9
10:29:16.369 CMD: Grp 0, Cmd 'SENSOR', Idx 34, Len 1, Pld 2, Data '2'
10:29:16.375 MQT: stat/scale/RESULT = {"Sensor34":"Load reference weight"}
10:29:20.101 MQT: tele/scale/SENSOR = {"Time":"2023-12-25T10:29:20","HX711":{"Weight":0.000,"WeightRaw":-129759,"AbsRaw":-129759}}
10:29:27.297 WIF: Checking connection...
10:29:30.049 MQT: tele/scale/SENSOR = {"Time":"2023-12-25T10:29:30","HX711":{"Weight":0.000,"WeightRaw":-117265,"AbsRaw":-117265}}
10:29:32.356 MQT: stat/scale/RESULT = {"Sensor34":"Calibration failed"}
```
The reference weight is 2324g. When I am not calibrating setting that weight changes WeightRaw from -199 to -174, but when I start calibrating and set the same weight as before  WeightRaw changes from -199 to -117265. This happens every time I try calibrating and with any reference weight that I tried so far. My guess is that is the reason for the failed calibration.
I am using Tasmota 13.3.0(sensors) with 4 load sells, HX711 and NodeMCU ESP 8266
What could be the problem?
Thank you

**Reply by weisskopfjens** [reply]
I have the same problem

**Comment by Quest24** [comment]
I have the same problem. Calibration failed and no information about the reason. Anyone?
I have the HX711 on 5 volt and the ESP 8266 on 3.3 volt and a level converter for DT and SCK.

**Comment by Quest24** [comment]
I my case there were 2 problems. Look at the absraw weight in the console (set telemetry period to 10 seconds while debugging)  
1. the raw weight was decreasing when I added weight. You can solve this by swap wires on A- en A+
2. the absraw weight was to high. I solved this by adding a trimpot (adjustable resistor) of 200k  on one of the loadcells between red and white ( or black depending on the problem).  adjust the potmeter so that the absraw weight is more to zero and positive.
